src/main/java/com/uniqueAuction/domain/product/service/ProductService.java

```diff
@@ -27,12 +25,12 @@ public Product productFindById(Long id) {
                 .orElseThrow(() -> new ProductException(NOT_FOUND_PRODUCT));
     }
 
-    public void updateProduct(Long id, ProductUpdateRequest productUpdateRequest) {
+    public Product updateProduct(Long id, Product updateProduct) {
```

Comment:
1.
파라메터의 변수명으로 `updateProduct`로 되어있는데요
이는 함수명으로는 적절하나 변수명으로는 적절하지 않은것 같습니다.
단순히 product 로도 충분합니다.

2. 
일반적으로 update의경우 Product entity에 id값이 포함되어있으니 별도의 id값을  따로 받을 필요가 없습니다.
따로 받으신 이유가 있나요?

```suggestion
    public Product updateProduct(Product product) {
```